src/components/cell-control/autocompleter.jsx

```diff
@@ -17,6 +18,11 @@ const Autocompleter = forwardRef(function Autocompleter(props, ref) {
     >
       <AutocompleterControl
         apiEndpoint={props.apiEndpoint}
+        classNames={{
```

Comment:
Do we want to provide a way to override these styles? Or do we wait until a need shows itself?

Reply:
I'd like to wait! I'm thinking the "cell" should look pretty good by itself. But I'll check it out during the integration with bigmaven.